streams/src/test/java/org/apache/kafka/streams/tests/RelationalSmokeTest.java

```diff
@@ -637,13 +637,14 @@ public static Topology getTopology() {
         public static Properties getConfig(final String broker,
                                            final String application,
                                            final String id,
+                                           final String processingGuarantee,
```

Comment:
Reading from the comment, the RelationalSmokeTest has nothing to do with EOS, why do we need to expand this test? cc @vvcephei

Reply:
The test had EOS enabled. So I extended it to now run with both EOS versions.

Reply:
I think it worth adding both eos mode for this test --- if we only have this test with eos-beta there may be some regressions not captured.

In the future, when we deprecate eos-alpha in some release version that requires broker > 2.5 we can remove eos-alpha parameters in all these.